web/templates/base/_items/vulnerability_tab_content.html

```diff
@@ -21,6 +21,17 @@
 			<div class="card-body">
 				<div class="row">
 					<div class="col-12">
```

Comment:
Could you add before the delete button, a green status button to permit mass update of the vuln status

Reply:
Sure on it

Reply:
Resolved